scripts/hv_stats/hv_stats.py

```diff
@@ -1,2612 +1,2617 @@
+            p2.yaxis.major_label_text_font_size = '12pt'
+            p2.legend.location = "bottom_right"
+            p2.legend.background_fill_alpha = 0.3
+            p2.border_fill_color = "whitesmoke"
+            p2.x_range.range_padding = 0.05
+
+            grid = gridplot([[p, p2]], sizing_mode='stretch_both')# width_policy='max', height_policy='max')#,plot_width=1200, plot_height=1000, sizing_mode='scale_width')#, plot_width=250, plot_height=250)
+            panel = Panel(child=grid, title=axis_type)
+            panels.append(panel)
+        tabs = Tabs(tabs=panels)
+#         show(tabs)
+        panel_obs = Panel(child=tabs, title=name.replace(observatory+' ',''))
+        panels_obs.append(panel_obs)
+    tabs_obs = Tabs(tabs=panels_obs)
+#     show(tabs_obs)
+#     break
+    save(tabs_obs, filename='./%s/%s_histogram.html'%(directory, observatory), title="Histogram and Cumulative distribution for %s"%observatory)
+print("Histograms and cumulative distribution plots completed.")
+
+
+# # POSTER PLOTS
+
+# # Helioviewer Movie length histogram
+
+# In[21]:
+
+
+print("### Helioviewer Movies' Length histogram ###")
+print("Starting SQL query in movies table of hv database...")
+start_time = time.time()
+hv={}
+# query="SELECT dataSourceString FROM movies"
+# query = "SELECT DATE_FORMAT(reqStartDate, '%1980-%m-%d %H:%i:%S') AS reqStartDate, timestamp, reqEndDate, startDate, endDate, dataSourceString FROM movies"
+# date_format(reqStartDate, '%Y-%m-%d %H:%i:%s') AS REQ_START, date_format(reqEndDate, '%Y-%m-%d %H:%i:%s')
+# query = "SELECT reqStartDate, reqEndDate, dataSourceString, eventSourceString, numFrames, frameRate, maxFrames, timestamp as date, TIMESTAMPDIFF(second, reqStartDate, reqEndDate) AS reqDuration, TIMESTAMPDIFF(second, startDate, endDate) AS genDuration FROM movies WHERE reqEndDate!='None' AND reqStartDate!='None' AND startDate!='None' AND endDate!='None';"
+# today = datetime.datetime.now().strftime('%Y-%m-%d')
+query = "SELECT reqStartDate, reqEndDate, dataSourceString, eventSourceString, numFrames, frameRate, maxFrames, timestamp as date, TIMESTAMPDIFF(second, reqStartDate, reqEndDate) AS reqDuration, TIMESTAMPDIFF(second, startDate, endDate) AS genDuration FROM movies WHERE reqStartDate IS NOT NULL AND reqEndDate IS NOT NULL AND startDate IS NOT NULL AND endDate IS NOT NULL AND timestamp IS NOT NULL;"
+# query = "SELECT ROUND(TIMESTAMPDIFF(second, reqStartDate, reqEndDate)/60/60/24, 3) AS reqDuration, ROUND(TIMESTAMPDIFF(second, startDate, endDate)/60/60/24, 3) AS genDuration FROM movies;"
+hv['hv_movies'] = sql_query(query)
+print("Query completed in %d seconds."%(time.time()-start_time))
+
+
+# In[64]:
+
+
+df = hv['hv_movies'].copy()
+
+# Timedelta.max.value is in nanoseconds. Divide by 10^9 to get the number of seconds supported
+max_timedelta_seconds = pd.Timedelta.max.value / 1e9
+# Set unsupported values to nan
+df['reqDuration'].loc[df['reqDuration'] > max_timedelta_seconds] = np.nan
```

Comment:
These are the only lines added. The giant change is from changing from windows line endings to unix line endings.